EVENTS.md

```diff
@@ -9,67 +9,57 @@ key.
     "date": "YYYY-MM-DD HH:mm:ss",
     "channel": "system",
     "values": {
-      "key": "value"
+      "key": <value>
     }
 }
 ```
 
 ## Values
 
+| Event                | Key       | Type     | Example value              | Description                                            |
+|----------------------|-----------|----------|----------------------------|--------------------------------------------------------|
+| **points_gained**    |           |          |                            | Attribute or skill points gained                       |
+|                      | name      | string   | `Agility`                  | Name                                                   |
+|                      | value     | float    | `50.8`                     | Points value                                           |
+|                      | type      | string   | `skill`                    | Type of point `skill` or `attribute`                   |
+| **damage_inflicted** |           |          |                            | Player inflicted damage                                |
+|                      | amount    | float    | `50.0`                     | Amount of damage inflicted                             |
+|                      | critical  | bool     | `true`                     | Critical hit                                           |
+| **damage_taken**     |           |          |                            | Player took damage                                     |
+|                      | amount    | float    | `50.0`                     | Amount of damage taken                                 |
+|                      | critical  | bool     | `false`                    | Critical hit                                           |
+| **enemy_evade**      |           |          |                            | Enemy evaded damage                                    |
+|                      | reason    | string   | `dodge`                    | Possible values: `dodge`, `evade`, `jam` or `miss`     |
```

Comment:
I am not sure, for the miss values. Is not that the enemy missed the player in this case

Reply:
It would be the equivalent to the old `enemy_miss` aka "Enemy missed an attack".
Maybe it makes more sense to move them out into separate events `player_miss` and `enemy_miss` again.

Reply:
It could be. Since they are quite different in meaning.
Player missed an attack does not mean a player evaded and vice versa for the mob